x/confidentialtransfers/client/cli/tx.go

```diff
@@ -173,3 +168,285 @@ func makeCloseAccountCmd(cmd *cobra.Command, args []string) error {
 
 	return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
 }
+
+// NewTransferTxCmd returns a CLI command handler for creating a MsgTransfer transaction.
+func NewTransferTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "transfer [denom] [to_address] [amount] [flags]",
+		Short: "Make a confidential transfer to another address",
+		Long: `Transfer command create a confidential transfer of the specified amount of the specified denomination to the specified address. 
+        passed in .`,
+		Args: cobra.ExactArgs(3),
+		RunE: makeTransferCmd,
+	}
+
+	flags.AddTxFlagsToCmd(cmd)
+	cmd.Flags().StringSlice("auditors", []string{}, "List of auditors")
+
+	return cmd
+}
+
+func makeTransferCmd(cmd *cobra.Command, args []string) error {
+	clientCtx, err := client.GetClientTxContext(cmd)
+	if err != nil {
+		return err
+	}
+
+	queryClientCtx, err := client.GetClientQueryContext(cmd)
+	if err != nil {
+		return err
+	}
+
+	queryClient := types.NewQueryClient(queryClientCtx)
+
+	privKey, err := getPrivateKey(cmd)
+	if err != nil {
+		return err
+	}
+
+	fromAddress := clientCtx.GetFromAddress().String()
+	denom := args[0]
+	toAddress := args[1]
```

Comment:
we may want to validate inputs. I think I skipped those checks in commands I implemented, and we may need to revisit here and other places